compiler-core/src/ast.rs

```diff
@@ -247,6 +247,18 @@ impl<A> RecordConstructor<A> {
     pub fn put_doc(&mut self, new_doc: (u32, EcoString)) {
         self.documentation = Some(new_doc);
     }
+
+    pub fn argument_labels(&self) -> Vec<&str> {
```

Comment:
This method doesn't seem generally useful as it discards non-labelled arguments, so you can only safely use it when you know for a fact that all the arguments are labelled. Let's inline this at the place it is used.